ms365_accountcreator/templates/index.html

```diff
@@ -73,20 +73,50 @@ <h1>{{ gettext('Account Registration for Microsoft Teams for University of Stutt
     </div>
 
     <script>
-        function emailSubmit(event) {
-            event.preventDefault();
+        const url_api_endpoint_email_verification = "{{url_api_endpoint_email_verification}}"
+        const url_api_endpoint_account_creation = "{{url_api_endpoint_account_creation}}"
 
-            document.getElementById('step-1').classList.add('done');
+        function postData(url = '', data = {}) {
+            return fetch(url, {
+                method: 'POST',
+                mode: 'cors',
+                cache: 'no-cache',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify(data)
+            });
+        }
 
-            const step2 = document.getElementById('step-2');
-            step2.classList.remove('inactive');
-            step2.setAttribute('open', '');
+        function emailSubmit(event) {
+            event.preventDefault();
+            if(! document.getElementById('input-consent-mail').checked) {
+                alert("{{ gettext('Consent needed') }}")
+                return false;
+            }
 
+            let email = document.getElementById('input-email').value
+            postData(url_api_endpoint_email_verification, {'email': email}).then(response => {response.json().then(
+                response_data => {
+                    if (response_data.valid) {
+                        document.getElementById('step-1').classList.add('done');
+                        const step2 = document.getElementById('step-2');
+                        step2.classList.remove('inactive');
+                        step2.setAttribute('open', '');
+                    } else {
+                        alert(response_data.reason)
+                    }
+                })}
+            );
             return false;
         }
 
         function consentSubmit(event) {
             event.preventDefault();
+            if(! (document.getElementById('input-consent-mail').checked && document.getElementById('input-consent-teams').checked)) {
```

Comment:
use extra variables for better readability (e.g. `const mailConsent = ...`)

Reply:
```suggestion
            const mailConsent = document.getElementById('input-consent-mail').checked;
            const teamsConsent = document.getElementById('input-consent-teams').checked;
            if(!(mailConsent && teamsConsent)) {
```